### PR TITLE
feat: keep well variables constant during poromechanics initizaliation

### PR DESCRIPTION
**Pull request by paveltomin**
Useful for sequential, removes the need to create well rate tables with zero rate for t < 0

### CODE REVIEW

File: src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseBase.cpp
Comment (tjb-ltk) [inline src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseBase.cpp:1]: In the event section if there are gaps in time a solver isn't called and if you setup a coupled solver but only define the reservoir solver in the events section. coupled solver isn't called.. I could be wrong on this... but if not, why isn't the problem decoupled by setting up different solvers and triggers via events..

Comment (paveltomin) [reply src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseBase.cpp:1]: one can not run pure solid mechanics solver/kernels for initialization because they don't have pressure contribution in them

**Comment by paveltomin** [comment]
need review please

File: src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.cpp [hunk 1]
```diff
@@ -1450,13 +1450,13 @@ void CompositionalMultiphaseBase::applyBoundaryConditions( real64 const time_n,
...
 
-  if( m_keepFlowVariablesConstantDuringInitStep )
+  if( m_keepVariablesConstantDuringInitStep )
   {
     // this function is going to force the current flow state to be constant during the time step
     // this is used when the poromechanics solver is performing the stress initialization
     // TODO: in the future, a dedicated poromechanics kernel should eliminate the flow vars to construct a reduced system
     //       which will remove the need for this brittle passing aroung of flag
-    keepFlowVariablesConstantDuringInitStep( time_n, dt, dofManager, domain, localMatrix.toViewConstSizes(), localRhs.toView() );
+    keepVariablesConstantDuringInitStep( time_n, dt, dofManager, domain, localMatrix.toViewConstSizes(), localRhs.toView() );
```
Comment (CusiniM) [inline src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.cpp:1460]: why the change of name? The goal is still to keep flow variables constant while updating displacement, isn't it?

File: src/coreComponents/physicsSolvers/fluidFlow/wells/WellSolverBase.cpp [hunk 2]
```diff
@@ -198,20 +198,23 @@ void WellSolverBase::assembleSystem( real64 const time,
...
 
-  // then assemble the volume balance equations
-  assembleVolumeBalanceTerms( domain, dofManager, localMatrix, localRhs );
+    // then assemble the volume balance equations
+    assembleVolumeBalanceTerms( domain, dofManager, localMatrix, localRhs );
 
-  // then assemble the pressure relations between well elements
-  assemblePressureRelations( time, dt, domain, dofManager, localMatrix, localRhs );
+    // then assemble the pressure relations between well elements
+    assemblePressureRelations( time, dt, domain, dofManager, localMatrix, localRhs );
```
Comment (CusiniM) [inline src/coreComponents/physicsSolvers/fluidFlow/wells/WellSolverBase.cpp]: some weird whitespace. Uncrustify should fix it.

**Review by CusiniM — APPROVED** [review]

File: src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.cpp (hunk 1 above)
Comment (paveltomin) [reply src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.cpp:1460]: just for some symmetry with wells, prefer functions to be called the same so it is easier to find

File: src/coreComponents/physicsSolvers/fluidFlow/wells/WellSolverBase.cpp (hunk 2 above)
Comment (paveltomin) [reply src/coreComponents/physicsSolvers/fluidFlow/wells/WellSolverBase.cpp]: i did run Uncrustify and the code style check is good
not sure what git is showing here

**Review by jhuang2601 — APPROVED** [review]
